src/iframe-apprtc/index.html

```diff
@@ -14,11 +14,16 @@
   <meta http-equiv="X-UA-Compatible" content="chrome=1"/>
 </head>
 <body>
-  AppRTC in an &lt;iframe&gt; element: To be deprecated in M64 (~ Jan 2018) <br>
-  <iframe width="50%" height="600px" src="//appr.tc"></iframe>
-  <hr>
-  To test website still uses cross-origin iframes <br>
-  <iframe width="50%" height="600px" src="//appr.tc" allow="geolocation; microphone; camera"></iframe>
-  <hr>
+<table style="border:1px solid #000; witdh:100%;">
```

Comment:
width is misspelled.

Reply:
Also, please put all style attributes in the main css [file](https://github.com/webrtc/test-pages/tree/gh-pages/src/css). Make sure it does not conflict with other pages using the css file.